src/lst/LSTDistributionGraph.jsx

```diff
@@ -189,80 +174,75 @@ export default function LSTDistributionGraph({
         return { timestamp: d.timestamp, value };
       }),
       brushPosition: {
-        start: scaleBrushDate(data[0]?.timestamp),
-        end: scaleBrushDate(data[data.length - 1]?.timestamp)
+        start: scaleBrushDate(points[points.length - 1 - 90].timestamp),
+        end: scaleBrushDate(points[points.length - 1].timestamp)
       },
+      filterPoints: points.slice(-90),
```

Comment:
```suggestion
      filteredPoints: points.slice(-90),
```
This will fix the brush and graph timeline mismatch on page load as well